packages/ts-migrate-plugins/tests/src/explicit-any.test.ts

```diff
@@ -1,26 +1,26 @@
 import ts from 'typescript';
-import { mockPluginParams, mockDiagnostic } from '../test-utils';
+import { mockPluginParams, mockDiagnostic, realPluginParams } from '../test-utils';
 import explicitAnyPlugin from '../../src/plugins/explicit-any';
 
 describe('explicit-any plugin', () => {
   it('adds explicit any', async () => {
-    const text = `import specifier1 from '../source';
+    const text = `let somePromise: any;
 somePromise.then(res1 => res1.default || res1);
 somePromise.then((res2) => res2.default || res2);
+let someArray: any;
 someArray.forEach(({ arg1, arg2 }) => {});
-const { dest1, dest2 } = obj;
```

Comment:
Destructuring properties into variables like this never seems to trigger error a "no implicit any" error. (TS7031 in the original test cases.)

[TS playground examples](https://www.typescriptlang.org/play?module=5&ts=4.1.2#code/N4KABGA2CmAuYHsBGArAhALjMAJtAzrAIxYCGAdgJ4A0YehATGVQL4Dc4YAxguYdnQLFa9WAzAswAXkSoOLEKAgx4yFM0ocIPPvGCDCREUPGSZa+Ys4rZ6sAFdyAa3IIA7uS3de-faKMGYhLStvJAA)